README.md

```diff
@@ -26,8 +26,8 @@ The following features have been implemented:
  - Record `perf` data of the whole system, TiDB modules, or specific process. (disabled by default)
  - Check the file size of TiDB modules' data directories. (enabled by default)
  - Collect and save log files of TiDB modules or system. (disabled by default)
- - Collect ans save various config files of system. (disabled by default)
- - Query PD API to collect runtime information of TiDB cluster. (enabled by default)
+ - Collect ans save various configuration files of system. (disabled by default)
```

Comment:
Collect **and** save various configuration files of the system. (disabled by default)